.github/workflows/deploy.yml

```diff
@@ -48,10 +48,20 @@ jobs:
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v1
 
-      - name: Build and push the image
+      - name: Build and push the Alpine runner-terraform image
         uses: docker/build-push-action@v2
         with:
+          file: ./alpine/Dockerfile
           push: ${{ github.ref == 'refs/heads/main' || github.ref == 'refs/heads/future' }}
           tags: |
             ${{ secrets.PUBLIC_RUNNER_TERRAFORM_ECR_REPOSITORY_URL }}:${{ github.ref == 'refs/heads/main' && 'latest' || 'future' }}
             ghcr.io/spacelift-io/runner-terraform:${{ github.ref == 'refs/heads/main' && 'latest' || 'future' }}
+
+      - name: Build and push the Debian runner-terraform image
+        uses: docker/build-push-action@v2
+        with:
+          file: ./debian/Dockerfile
+          push: ${{ github.ref == 'refs/heads/main' || github.ref == 'refs/heads/future' }}
+          tags: |
+            ${{ secrets.PUBLIC_RUNNER_TERRAFORM_ECR_REPOSITORY_URL }}:debian
+            ghcr.io/spacelift-io/runner-terraform:debian
```

Comment:
Can you parameterize the tags like we're doing here: https://github.com/spacelift-io/runner-terraform/pull/15/files#diff-28802fbf11c83a2eee09623fb192785e7ca92a3f40602a517c011b947a1822d3R58

Maybe make the future branch create a `debian-future` tag?

I don't think it matters for the ECR repo since we have a separate repo for dev, but it matters for the ghcr repo since there's only one. Otherwise any pushes to `future` would overwrite the release version of the image with the development version.

Just FYI in case you haven't come across it, the ghcr repo is just there as a fallback in case ECR fails.

Reply:
@adamconnelly I intentionally didn't tag the image this way as we were going to have `debian` as a separate tag on `runner-image` rather than a separate image altogether. The only way i'd be able to use the existing versioning information is if I was to append something to debian, such as `runner-terraform:debian-$versionInfo` - which I'd be okay with as well - need to figure out which path we are going to go with.

Reply:
Yeah, I'm fine with `runner-image:debian`. It's just that because of the branching model of this repo, your current change will cause development changes that shouldn't be released to be released as the production image in GHCR.

Reply:
@adamconnelly Thanks I understand now, just resolved this.